jenkins/gradle/gradle-check.jenkinsfile

```diff
@@ -0,0 +1,123 @@
+lib = library(identifier: 'jenkins@20211123', retriever: legacySCM(scm))
+
+pipeline {
+    options {
+            timeout(time: 2, unit: 'HOURS')
+    }
+    // gradle check have a lot of issues running on containers
+    // Therefore, we directly run it on the agent node
+    agent {
+        node {
+            // Must use Ubuntu agent with 1 executor or gradle check will show a lot of java-related errors
+            label 'Jenkins-Agent-Ubuntu2004-X64-c54xlarge-Single-Host'
```

Comment:
Can we link the issue here in the comment: https://github.com/opensearch-project/OpenSearch/issues/3674? Later we can run gradle check in the container.

Reply:
We have decided to not run gradle check in container anymore.
This is also not fully related to container, but to run gradle check on nothing but Ubuntu.
AL2 container will fail and AL2 EC2 will also fail. OS that is not Ubuntu will fail.